azure-pipelines/variables.yml

```
@@ -0,0 +1,16 @@
+variables:
+  windows_image: 'windows-2019'
+  linux_image: 'ubuntu-18.04'
+  mac_image: 'macOS-10.15'
+  CACHE_VCPKG_KEY: v4 | vcpkg-root
+  VCPKG_CMAKE: "$(Build.SourcesDirectory)/vcpkg/scripts/buildsystems/vcpkg.cmake"
+  #ARCH:
+  #TRIPLET:
+  #VCVARS:
+  SDL2_STATIC: ON
+  BUILD_TESTING: ON
+  BUILD_SHARED_LIBS: ON
+  BUILD_EXAMPLES: ON
+  BUiLD_SNAPSHOT: OFF
```

Azure pipeline is case insensitive...
```suggestion
  BUILD_SNAPSHOT: OFF
```